src/OmniSharp.Abstractions/Endpoints.cs

```diff
@@ -64,5 +59,9 @@ public static class V2
             public const string GetCodeActions = "/v2/getcodeactions";
             public const string RunCodeAction = "/v2/runcodeaction";
         }
+        
+        public const string GetTestStartInfo = "/getteststartinfo";
```

Comment:
will call it `/v2/getteststartinfo` since the function of the endpoint is narrowly defined.